tripal/tripal
# Chado fix for box_range

# Bug Fix

### Issue #1746 


### Tripal Version:  4.x

## Description


There is a bug in the `box_range` function of Chado that doesn't cast a 0 value passed to `create_point` to a bigint.  This causes the an error when cloning a Chado schema or deleting a feature.   

![image](https://github.com/tripal/tripal/assets/1719352/fdce5134-c0b3-498d-92a5-bea492e53f43)

This PR applies a change to the Chado SQL file in Tripal that includes a fix in the Chado repo here: https://github.com/GMOD/Chado/pull/105/files


## Testing?



1. Install a fresh version of Tripal using this branch. 
2. Install Chado and then try to clone it. Cloning should work just fine.  
3. Create an Organism content type, then a Gene content type using the Tripal interface.  Next use your favorite SQL tool to manually delete the gene from the feature table.  It should delete without a problem.
